src/content/ui/interactivity/focus.md

```diff
@@ -3,179 +3,226 @@ title: Understanding Flutter's keyboard focus system
 description: How to use the focus system in your Flutter app.
 ---
 
-This article explains how to control where keyboard input is directed. If you
-are implementing an application that uses a physical keyboard, such as most
-desktop and web applications, this page is for you. If your app won't be used
-with a physical keyboard, you can skip this.
+This article explains how to direct keyboard input.
+If your app uses a physical keyboard, like desktop and web apps,
+keep reading. If your app won't use a physical keyboard,
+you can skip this guide.
 
 ## Overview
 
-Flutter comes with a focus system that directs the keyboard input to a
-particular part of an application. In order to do this, users "focus" the input
-onto that part of an application by tapping or clicking the desired UI element.
-Once that happens, text entered with the keyboard flows to that part of the
-application until the focus moves to another part of the application.  Focus can
-also be moved by pressing a particular keyboard shortcut, which is typically
-bound to <kbd>Tab</kbd>, so it is sometimes called "tab traversal".
+Flutter includes a focus system that directs keyboard input
+to a particular part of an app.
+To "focus" the input to part of an app, app users can take action
+depending on their input method.
 
-This page explores the APIs used to perform these operations on a Flutter
-application, and how the focus system works. We have noticed that there is some
-confusion among developers about how to define and use [`FocusNode`][] objects.
-If that describes your experience, skip ahead to the [best practices for
-creating `FocusNode` objects](#best-practices-for-creating-focusnode-objects).
+* To change focus on touch interfaces,
+  app users tap or click on the desired UI element.
 
-### Focus use cases
+* To change focus on traditional interfaces,
+  app users can use the mouse to click on the desired UI element
+  or press a keyboard shortcut to move focus.
 
-Some examples of situations where you might need to know how to use the focus
-system:
+Once in focus, text entered with the keyboard flows to that element.
+Text input could control objects or display text in a text field.
+This continues until the focus moves to another element in the app.
 
-- [Receiving/handling key events](#key-events)
-- [Implementing a custom component that needs to be focusable](#focus-widget)
-- [Receiving notifications when the focus changes](#change-notifications)
-- [Changing or defining the "tab order" of focus traversal in an application](#focustraversalpolicy)
-- [Defining groups of controls that should be traversed together](#focustraversalgroup-widget)
-- [Preventing some controls in an application from being focusable](#controlling-what-gets-focus)
+:::secondary What you'll learn
```

Comment:
Why did you replace the list of focus use cases with "what you'll learn"? This isn't a tutorial. I put those use cases there for a VERY SPECIFIC reason. I wanted to make it very clear what situations might require fiddling with the focus. The "what you'll learn" doesn't convey the same info. And since we've already said that this guide doesn't apply to everyone, why treat it like a general tutorial?

Reply:
The list of use cases that Greg added in his [initial PR](https://github.com/flutter/website/pull/6243/commits/944b5f33764406ea46efbf7374150a2d8a4eba8c#diff-a7a3d5017af0d2e57bf7203482edf9adbfbcd333fcda4ce15ad4540832affd5aR29-R39) read as lacking context, so they were moved to [When to use the focus system](https://flutter-docs-prod--pr10749-fix-10168-copy-qvo61uju.web.app/ui/interactivity/focus#when-to-use-the-focus-system).